android/app/src/main/java/app/organicmaps/sdk/routing/ResultCodes.java

```diff
@@ -0,0 +1,23 @@
+package app.organicmaps.sdk.routing;
+
+public interface ResultCodes
+{
+  // Codes correspond to native routing::RouterResultCode in routing/routing_callbacks.hpp
+  int NO_ERROR = 0;
```

Comment:
Why not static final? Are they intended to be overridden?

Reply:
answered above